editor/src/components/editor/store/dispatch-strategies.spec.tsx

```diff
@@ -16,9 +16,9 @@ import {
   interactionCancel,
   interactionHardReset,
   interactionStart,
-  interactionStrategyChangeStacked,
+  handleStrategyChangeStacked,
```

Comment:
these new "inner" functions seem to be unused here. I think it'd make the code nicer if they wouldn't even be exported from `dispatch-strategies.tsx`!

Reply:
true, fixed in 830607b4fd239aa890d919a32b8ef6d3eb7887da